app/frontend/src/aquifers/components/Form.vue

```diff
@@ -245,6 +245,58 @@
       </b-col>
     </b-row>
 
+    <b-row class="mt-4"
```

Comment:
We should consider making this its own component, but it does complicate things a bit.